.kokoro/release/stage.sh

```diff
@@ -39,11 +39,19 @@ mvn clean deploy -B \
   -Dgpg.homedir=${GPG_HOMEDIR} \
   -P release
 
+echo "Finding nexus-staging"
+find . -name "nexus-staging"
+
+echo "Done"
+echo "Finding staging"
+find . -name "staging"
+echo  "Done"
+
```

Comment:
These lines clarify how the two mvn commands communicate staging repository IDs.

It didn't find any:

```
Finding nexus-staging
Done
Finding staging
Done
```